woocommerce/wc-api-dev
# Hotfix: Don't auto-enable the PayPal payment method

Similar to #33, this makes it so the PayPal method is not auto-enabled by default. This filter only overrides the defaults that are used when no settings are found. 

To Test:
* Install this branch/plugin over your existing wc-api-dev copy.
* Unless you just set-up a fresh store, you will need to remove your settings for the PayPal payment method. What I did for my AT site was temporarily add `delete_option( 'woocommerce_paypal_settings' );`.
* View the checkout page and make sure the PayPal payment method is not displayed.
* Go to `http://calypso.localhost:3000/store/settings/:site` and enable the PayPal payment method.
* View the checkout page and make sure the PayPal payment method is displayed.
